.pre-commit-config.yaml

```diff
@@ -11,7 +11,11 @@ repos:
     rev: ""
     hooks:
       - id: flake8
-        exclude: legacy/
+        exclude: >
+            (?x)^(
+                legacy/|
+                backend/emails/contents.py
```

Comment:
Huh, wondering why we had to do this?

Reply:
I had to do it because the flake8 linter checks that the lines are not more than 80 characters long
The `contents.py` file is the one that includes the emails content. Its lines are longer than the expected, and didn't make sense to format them since aren't code